src/FreeAge/server/map.cpp

```diff
@@ -47,8 +48,10 @@ ServerMap::~ServerMap() {
 
 
 
-void ServerMap::GenerateRandomMap(int playerCount, int seed) {
+void ServerMap::GenerateRandomMap(Game* game, int seed) {
```

Comment:
I forgot about this... I basically pass the whole game (in the future could be just a struct with the players and the map settings). Players are need because the civ can affect the starting objects (number of villagers, starting scout unit, (possibly more things?))